assignments/json_parser_test.go

```diff
@@ -0,0 +1,57 @@
+package assignments
```

Comment:
Consider if this test should be removed or integrated into `assignments_parser_test.go`.